src/python/zquantum/core/circuit/conversions/symbolic_expressions_test.py

```diff
@@ -268,6 +268,22 @@ def test_sympy_function_calls_are_converted_to_function_call_object_with_appropr
         (
             sympy.tan(sympy.Symbol("theta")),
             quilatom.quil_sin(quil.Parameter("theta")) / quilatom.quil_cos(quil.Parameter("theta"))
+        ),
+        (
+            2 ** sympy.Symbol("x"),
+            2 ** quil.Parameter("x")
+        ),
+        (
+            sympy.Symbol("y") ** sympy.Symbol("x"),
+            quil.Parameter("y") ** quil.Parameter("x")
+        ),
+        (
+            sympy.Symbol("x") ** 2,
+            quil.Parameter("x") ** 2
+        ),
+        (
+            2 ** sympy.Symbol("x"),
+            2 ** quil.Parameter("x")
```

Comment:
This seems to be a repeat of 273->274 no?

Reply:
Nive catch. I removed the duplicate.